WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -16067,6 +16092,7 @@
 				2FAE97090E33B21600CA8540 /* defaultPostTemplate_old.html in Resources */,
 				3236F77624ABB7770088E8F3 /* ReaderInterestsCollectionViewCell.xib in Resources */,
 				9A19D441236C7C7500D393E5 /* StatsGhostTopHeaderCell.xib in Resources */,
+				B02D1ECF2835348900F20359 /* pride-'22-icon-app-76x76@2x.png in Resources */,
```

Comment:
> pride-'22-icon-app-76x76@2x.png

👋 @frosty, the name of the files here includes a single quote `'` because we wanted to name the icon `Pride '22`. In my testing, the value of [`defaultIconName`](https://github.com/wordpress-mobile/WordPress-iOS/blob/e56bbded77a8cca0e39697a24008ea348f260120/WordPress/Classes/Utility/AppIcon.swift#L96) needed to coincide with the file name (according to [the logic here, which lowercases the display name and replaces spaces with hyphens](https://github.com/wordpress-mobile/WordPress-iOS/blob/08d12dfc094177f648f4330226cba9df56a1306c/WordPress/Classes/Utility/AppIcon.swift#L21)).
Do you know if there there a better way to do this that I'm missing? I'm a bit wary of the quote in the filename somehow causing problems, but so far it seems to be not causing any.